test/fixtures/when.go

```diff
@@ -41,7 +41,7 @@ type When struct {
 	kubeClient     kubernetes.Interface
 
 	portForwarderStopChannels map[string]chan struct{}
-	// Key: vertex label selector
+	// Keys: vertex label selector
```

Comment:
```suggestion
	// Key: vertex label selector
```
this has nothing to do with keys in (keys, val) pair